iphone/Classes/GDataXMLNode.h

```diff
@@ -104,56 +103,61 @@ typedef NSUInteger GDataXMLNodeKind;
 	// We will fix up the node's namespace and name (and those of any children)
 	// later when adding the node to a tree with addChild: or addAttribute:.
 	// See fixUpNamespacesForNode:.
-	
-	xmlNodePtr xmlNode_; // may also be an xmlAttrPtr or xmlNsPtr
-	BOOL shouldFreeXMLNode_; // if yes, xmlNode_ will be free'd in dealloc
-	
+
+	xmlNodePtr xmlNode_;      // may also be an xmlAttrPtr or xmlNsPtr
+	BOOL shouldFreeXMLNode_;  // if yes, xmlNode_ will be free'd in dealloc
+
 	// cached values
-	NSString *cachedName_;
-	NSArray *cachedChildren_;
-	NSArray *cachedAttributes_;
+	NSString* cachedName_;
+	NSArray* cachedChildren_;
+	NSArray* cachedAttributes_;
 }
 
-+ (GDataXMLElement *)elementWithName:(NSString *)name;
-+ (GDataXMLElement *)elementWithName:(NSString *)name stringValue:(NSString *)value;
-+ (GDataXMLElement *)elementWithName:(NSString *)name URI:(NSString *)value;
++ (GDataXMLElement*)elementWithName:(NSString*)name;
++ (GDataXMLElement*)elementWithName:(NSString*)name
+                        stringValue:(NSString*)value;
++ (GDataXMLElement*)elementWithName:(NSString*)name URI:(NSString*)value;
 
-+ (id)attributeWithName:(NSString *)name stringValue:(NSString *)value;
-+ (id)attributeWithName:(NSString *)name URI:(NSString *)attributeURI stringValue:(NSString *)value;
++ (id)attributeWithName:(NSString*)name stringValue:(NSString*)value;
++ (id)attributeWithName:(NSString*)name
+                    URI:(NSString*)attributeURI
+            stringValue:(NSString*)value;
 
-+ (id)namespaceWithName:(NSString *)name stringValue:(NSString *)value;
++ (id)namespaceWithName:(NSString*)name stringValue:(NSString*)value;
 
-+ (id)textWithStringValue:(NSString *)value;
-+ (id)cDataSectionWithStringValue:(NSString *)value;
++ (id)textWithStringValue:(NSString*)value;
++ (id)cDataSectionWithStringValue:(NSString*)value;
 
-- (NSString *)stringValue;
-- (void)setStringValue:(NSString *)str;
+- (NSString*)stringValue;
```

Comment:
It is possible we are using an older version of clang-format. Perhaps updating to a newer compiled version would help.

It also seems to treat .h and .m files a bit differently.

How about http://objclean.com/index.php? It seems like it might work.